.github/workflows/ci.yml

```diff
@@ -77,4 +77,4 @@ jobs:
         if: steps.cache-dependencies.outputs.cache-hit != 'true'
 
       - name: Test
-        run: yarn test
+        run: yarn test:ember
```

Comment:
Running `test:ember` helps us continue to run linting and testing in parallel in CI (if this is what we want, instead of in serial).